source/macos/AdaptiveCards/AdaptiveCards/Utils/BundleUtils.swift

```diff
@@ -0,0 +1,22 @@
+import AppKit
+
+class BundleUtils {
+    static let bundleIdentifier = "com.test.test.AdaptiveCards"
+    
+    static func getImage(_ imageName: String, _ fileType: String) -> NSImage? {
+        guard let bundle = Bundle(identifier: bundleIdentifier),
```

Comment:
Why do we want to ignore the second param. Let it be there to make it readable and verbose enough while calling.
https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/CodingGuidelines/Articles/NamingMethods.html
```suggestion
    static func getImage(_ imageName: String, ofType type: String) -> NSImage? {
        guard let bundle = Bundle(identifier: bundleIdentifier),
```